### PR TITLE
Update wcpay suggestion UI in payment task

### LINKED ISSUE #33598: Payments Overview: Outdated design
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Payments overview is displaying some old version of the hero box.

### Expected behavior

It should be updated inline with the design in [Figma](https://www.figma.com/file/e428PXC0QyEwYqtX48lLq0/%5BWCPay%5D-Positioning-and-Conversion?node-id=12830%3A93773).

Screenshot 2022-06-24 at 17 56 55

New design was previously implemented as a test as a part of the [KYC Optimization Project](https://wcpay.wordpress.com/2022/02/15/project-thread-kyc-optimization/)

### Actual behavior

Outdated design of the Payments Overview hero box
Screenshot 2022-06-24 at 17 57 33

### Steps to reproduce

1. Onboard to WooCommerce from a WCPay eligible country **BUT** during onboarding opt out from installing WCPay
2. Go to the Payments Overview

### WordPress Environment

`
WordPress address (URL): https://rogermatticwoo.mystagingwebsite.com
Site address (URL): https://rogermatticwoo.mystagingwebsite.com
WC Version: 6.6.1
REST API Version: ✔ 6.6.1
WC Blocks Version: ✔ 7.6.2
Action Scheduler Version: ✔ 3.4.0
Log Directory Writable: ✔
WP Version: 6.0
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: ✔
`

### PR DESCRIPTION
**Pull request by chihsuan**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #33598.

This PR updates the wcpay suggestion UI in the payment task, and adds `<WCPayBanner />` and `<WCPayBenefits />` to `@woocommerce/onboarding` for the changes.

Before:

Screen Shot 2022-07-04 at 16 02 26

After:

Screen Shot 2022-07-04 at 15 48 42

I've also confirmed this change doesn't affect the banner in setting page:

Screen Shot 2022-07-04 at 16 01 51

### How to test the changes in this Pull Request:

1. Onboard to WooCommerce from a WCPay eligible country BUT during onboarding opt out from installing WCPay
2. Go to **Woocmmerce > Home**
3. Go to payment task
4. Observe that the UI matches the design (e428PXC0QyEwYqtX48lLq0-fi-12830%3A93773).

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [ ] Have you successfully run tests with your changes locally?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm changelog add --filter=<project>`?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

**Comment by botwoo** [comment]
:bar_chart: Test reports for this pull request have been published and are accessible through the following links:

- [REST API Test Report](https://woocommerce.github.io/woocommerce-test-reports/pr/33717/api/)
- E2E Test Report (TBC)

Latest commit referenced in the reports: `Add changelog` f37b63e5a12f181929d56a2ab9cd1f6db422fe7f
This comment will automatically be updated with the latest referenced commit when you push new changes to this pull request.

---

_Visit the [WooCommerce Test Reports](https://woocommerce.github.io/woocommerce-test-reports) homepage to view all published reports. See the [FAQs](https://github.com/woocommerce/woocommerce-test-reports/wiki/FAQs) page if you're having problems accessing them._

**Review by ilyasfoo — DISMISSED** [review]
Changes are looking good, @chihsuan! Thanks for trying to consolidate with WCPay banner.

Code-wise is fine, however, I've found a couple of nitpicky things for the UI:

1. It seems the image used in the payments page in figma differs from the WCPay banner. If you notice, the image we'd want has no right border and the right side appears like "cut-off" halfway:

svg


2. There are few minor kinks for the card below (left is from figma, right is current screen):

image

I'd be happy if you want to address it here or I can also make a follow-up for UI differences!

**Comment by chihsuan** [comment]
> I'd be happy if you want to address it here or I can also make a follow-up for UI differences!

@ilyasfoo Thanks for the review and catching them. ❤️ I think those are small changes. I'll address it in this PR.

**Comment by chihsuan** [comment]
@ilyasfoo I've updated the UI in ff4ba0c83445193bb50f2e1f34570df8fee545d0. Could you review it again, please?

File: packages/js/onboarding/src/components/WCPayBanner/WCPayBanner.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	min-width: 750px;
+	animation: isLoaded;
+	animation-duration: 250ms;
+
+	&.components-card {
+		box-shadow: none;
+		border: 1px solid $table-border;
+		border-radius: 2px;
+	}
+
```
Comment (ilyasfoo) [inline packages/js/onboarding/src/components/WCPayBanner/WCPayBanner.scss:12]: Extra space 😄 

```suggestion
```

**Review by ilyasfoo — DISMISSED** [review]
It's looking great, thanks! Super duper nitpicky still, I think if we can add `display: flex` to `woocommerce-recommended-payments-banner__image_container` would attach the image to the bottom of the container 😅 

image

But it's already good to be merged as well!

**Comment by chihsuan** [comment]
> It's looking great, thanks! Super duper nitpicky still, I think if we can add `display: flex` to `woocommerce-recommended-payments-banner__image_container` would attach the image to the bottom of the container 😅
> 
> image
> 
> But it's already good to be merged as well!

@ilyasfoo That looks perfect!  Updated in 973b0450581ebd3561af83d64a17cd3e93947bb3. Thanks. 😃

**Review by ilyasfoo — APPROVED** [review]
Looking great! Thanks! LGTM 🚢

**Pull request merged**